hash/msx1_cart.xml

```diff
@@ -13400,6 +12608,7 @@ kept for now until finding out what those bytes affect...
 		<publisher>Bandai</publisher>
 		<info name="serial" value="0201253 / BMX-004" />
 		<info name="alt_title" value="銀河漂流バイファム (Ginga Hyoryuu Vifam)" />
+		<info name="usage" value="Requires a MSX1 system."/>
 		<part name="cart" interface="msx_cart">
```

Comment:
Should this be “an MSX1 system”, since “MSX” is usually read as “em ess ex”?  It’s read as the initialism in Japanese, too (“エム・エス・エックス”, or “emu esu ekkusu”).